src/deploy/table_location.sql

```diff
@@ -0,0 +1,52 @@
+BEGIN;
+
+CREATE TABLE maevsi.location (
+  id            UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+
+  location_type CHAR(1) NOT NULL,
+  latitude      DOUBLE PRECISION NOT NULL,
+  longitude     DOUBLE PRECISION NOT NULL,
+
+  -- -- possible future extension using PostGIS
+  -- geom          GEOMETRY
```

Comment:
@sthelemann fyi: as PostGIS is enabled on the `maevsi` schema, this change will be needed
```suggestion
  -- geom          maevsi.GEOMETRY
```

Reply:
Why did you create the postgis extension on the maevsi schema and not on public?

Reply:
To make sure that all functionality is usable within the schema that is wrapped by Postgraphile, but if `public` works just as well, it can be added there too. I just always added extensions on the maevsi schema and continued to do so, can't remember if there was a relevant reason, but apparently I started with it once, so there might have been one.

Reply:
It's in public now.